src/components/InputTime/Dropdown/Dropdown.tsx

```diff
@@ -121,7 +121,9 @@ const Dropdown: React.FC<DropdownProps> = ({
     <EasyDropdown
       className={classNames(className, styles.Dropdown)}
       disabled={disabled}
-      isOpen={undefined}
+      isOpen={
+        undefined /* Suppresses typescript lint warning about missing prop */
+      }
```

Comment:
Why does `isOpen` need to be `undefined` instead of `false`?

Reply:
Added a comment to clarify this.
It's because of the TS component importing a JSX component. In this case, omitting `isOpen` was inaccurately throwing a missing prop error. But `EasyDropdown` is specifically looking for isOpen to be undefined - if it's undefined, the component uses an uncontrolled open state.

I could have also suppressed the TS warning by editing `EasyDropdown` to have 
```
const EasyDropdown = ({
  ...
  isOpen: controlledIsOpen = undefined,
  ...
}) => {
```

But that seemed just as weird. I figure porting EasyDropdown to TS will happen later, and then this will be a non-issue.